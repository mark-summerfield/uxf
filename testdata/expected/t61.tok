TClassBegin ttype=<img>
  Field Str("content")
  Field Str("image") vtype=bytes
TClassEnd
TClassBegin ttype=<p>
  Field Str("content")
TClassEnd
TClassBegin ttype=<b>
  Field Str("content")
TClassEnd
ListBegin
  TableBegin ttype=<p>
    Str("first para")
  TableEnd
  TableBegin ttype=<p>
    TableBegin ttype=<img>
      TableBegin ttype=<b>
        Str("bold caption")
      TableEnd
      Bytes( (:B445:) )
    TableEnd
  TableEnd
ListEnd
Eof
----------------------------------------
