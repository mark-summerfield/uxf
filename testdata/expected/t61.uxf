uxf 1.0
= b content
= img content image:bytes
= p content
[
  ( p <first para>)
  ( p (img (b <bold caption>) (:B445:)))
]
