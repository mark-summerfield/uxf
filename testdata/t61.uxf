uxf 1.0
= img content image:bytes
= p content
= b content
[
  (p <first para>)
  (p (img (b <bold caption>) (:B445:)))
]
