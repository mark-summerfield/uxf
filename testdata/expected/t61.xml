<?xml version="1.0" ?>
<uxf version="1.0">
  <ttypes>
    <ttype name="b">
      <field name="content"/>
    </ttype>
    <ttype name="img">
      <field name="content"/>
      <field name="image" vtype="bytes"/>
    </ttype>
    <ttype name="p">
      <field name="content"/>
    </ttype>
  </ttypes>
  <list>
    <table name="p">
      <row>
        <str>first para</str>
      </row>
    </table>
    <table name="p">
      <row>
        <table name="img">
          <row>
            <table name="b">
              <row>
                <str>bold caption</str>
              </row>
            </table>
            <bytes>B445</bytes>
          </row>
        </table>
      </row>
    </table>
  </list>
</uxf>
